internal/pkg/vm/actor/builtin/initactor/init.go

```diff
@@ -21,12 +24,26 @@ type Actor struct{}
 
 // State is the init actor's storage.
 type State struct {
-	Network string
+	Network    string
+	AddressMap cid.Cid `refmt:",omitempty"`
+	IdMap      cid.Cid `refmt:",omitempty"`
```

Comment:
As we discussed in person, the map from ID to actor address is unnecessary if the state tree is keyed by ID address (for non-account actors). I know it's in the spec but I think it will be removed.